app/scripts/views/mixins/signin-mixin.js

```diff
@@ -147,19 +148,29 @@ define(function (require, exports, module) {
 
     _engageSignInForm (event) {
       if (event && event.target) {
-        var target = this.$el.find(event.target).attr('id');
-
-        if (target === 'have-account') {
-          // if the user clicks on 'have-account' we count that as flow event instead of the 'engage' event.
-          // Details: https://github.com/mozilla/fxa-content-server/pull/4221
-          this.logEvent('flow.have-account');
-          return;
+        const target = this.$el.find(event.target).attr('id');
+
+        switch (target) {
+          case 'create-account':
+          case 'forgot-password':
+          case 'have-account':
+            // Clicking one of these links does not equate to
+            // engaging with the form itself. Emit different
+            // events for these cases.
+            this.logFlowEvent(target, this.viewName);
```

Comment:
To understand fully, the eventual event will be something like `flow.signin.create-account` if the user clicks a link?